### PR TITLE
CIV-5171 - Hearing Notice API tests

### PR DESCRIPTION
**Pull request by filippo-possenti-version1**
Hearing Notice API Tests for DJ and SDO journeys

### CODE REVIEW

File: e2e/tests/api_tests/defaultJudgments/api_dj_1v2_test.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ Scenario('Default Judgment claim SDO', async ({I, api}) => {
   if (config.runWAApiTest) {
     api.completeTaskByUser(config.judgeUserWithRegionId1, taskId);
   }
+  await api.scheduleHearing(config.hearingCenterAdminWithRegionId1, 'SMALL_CLAIMS');
```
Comment (mounikahmcts) [inline e2e/tests/api_tests/defaultJudgments/api_dj_1v2_test.js:40]: same here

File: e2e/tests/api_tests/defaultJudgments/api_dj_2v1_test.js [hunk 2]
```diff
@@ -38,6 +38,7 @@ Scenario('Default Judgment claim SDO', async ({I, api}) => {
   if (config.runWAApiTest) {
     api.completeTaskByUser(config.judgeUserWithRegionId1, taskId);
   }
+  await api.scheduleHearing(config.hearingCenterAdminWithRegionId1, 'FAST_TRACK_TRIAL');
```
Comment (mounikahmcts) [inline e2e/tests/api_tests/defaultJudgments/api_dj_2v1_test.js:41]: same here

File: e2e/tests/api_tests/defaultJudgments/api_dj_1v1_test.js [hunk 3]
```diff
@@ -37,6 +37,8 @@ Scenario('Default Judgment claim SDO', async ({I, api}) => {
   if (config.runWAApiTest) {
     api.completeTaskByUser(config.judgeUserWithRegionId1, taskId);
   }
+  await api.scheduleHearing(config.hearingCenterAdminWithRegionId1, 'OTHER');
```
[the remark this answers is not in the document]
Comment (m-meulendijks-v1) [reply e2e/tests/api_tests/defaultJudgments/api_dj_1v1_test.js]: Put in a separate scenario

File: e2e/tests/api_tests/defaultJudgments/api_dj_1v2_test.js (hunk 1 above)
Comment (m-meulendijks-v1) [reply e2e/tests/api_tests/defaultJudgments/api_dj_1v2_test.js]: Put in a separate scenario

File: e2e/tests/api_tests/defaultJudgments/api_dj_2v1_test.js (hunk 2 above)
Comment (m-meulendijks-v1) [reply e2e/tests/api_tests/defaultJudgments/api_dj_2v1_test.js]: Put in a separate scenario

**Review by m-meulendijks-v1 — APPROVED** [review]

**Comment by filippo-possenti-version1** [comment]
Worth noting that the new tests will need the previous one to be ran as well.
This is not ideal but luckily for us codeceptjs in our case is configured to split workers by "suite", so as long as they are in the same file the execution order is guaranteed.

**Review by mounikahmcts — APPROVED** [review]

**Pull request merged**